tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -975,6 +975,12 @@ show_techsupport/test_techsupport.py::test_techsupport:
 #######################################
 #####            snmp             #####
 #######################################
+snmp/test_snmp_memory.py::test_snmp_memory_load:
+  skip:
+    resason: "Test is flaky on some of the platforms"
+    conditions:
+      - https://github.com/sonic-net/sonic-mgmt/issues/8935
```

Comment:
Just want to double-confirm, test_snmp_memory_load case would be replaced with test_snmp_avail_memory_load, or we still need it, but need to do further analyse before remove skip, thanks.

Reply:
yes, that is right, further analysis is required to check if we can remove the test.
It seems like that test is not correct.
I have updated my findings in the issue. We increase memory load on the host and snmp free memory is obtained from SNMP docker which is not matching the host free memory.